packages/cli/src/lib/axe-test-urls.test.ts

```diff
@@ -1,60 +1,60 @@
-'use strict';
-
-const assert = require('chai').assert;
-const testPages = require('../lib/axe-test-urls');
+import 'mocha';
+import { assert } from 'chai';
+import testPages from './axe-test-urls';
 
 describe('testPages', function () {
   this.timeout(10000);
-  let config, mockDriver;
+  let config: any;
+  let mockDriver: any;
 
   beforeEach(() => {
     mockDriver = {
-      get: async arg => arg,
-      executeAsyncScript: async arg => arg,
-      executeScript: async arg => arg,
-      wait: async arg => arg,
+      get: async (arg: any) => arg,
```

Comment:
extract `async (arg: any) => arg` to be a variable & simply assign that to each prop here.